src/main/java/org/launchcode/liftoffproject/models/Chore.java

```diff
@@ -27,11 +34,17 @@ public class Chore extends AbstractEntity{
 
     public Chore(){}
 
-    public Chore(String name, String choreDescription, int rewardPoints, DayOfTheWeek dueDay) {
+
+
+
+    public Chore(String name, String choreDescription, int rewardPoints, DayOfTheWeek dueDay, String dayOfTheWeek, String detailedDescription, String supplies) {
         this.name = name;
         this.choreDescription = choreDescription;
         this.rewardPoints = rewardPoints;
         this.dueDay = dueDay;
+        this.dayOfTheWeek = dayOfTheWeek;
+        this.detailedDescription = detailedDescription;
```

Comment:
```suggestion
```
Remove detailedDescription from the constructor (I think you should only be able to access it from the Chore Details page).